File: A1/A1-026/A1-026.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input
    int a, b, c;
    cin >> a >> b >> c;
    // Count odd numbers
    int odd = (a % 2) + (b % 2) + (c % 2);
    // Output
    cout << "even " << 3 - odd << '\n';
    cout << "odd " << odd << '\n';
    return 0;
}

File: A1/A1-026/A1-026.py
# --------------------------------------------------
# File Name : A1-026.py
# Problem   : Count odd even
# Author    : Worralop Srichainont
# Date      : 2026-02-08
# --------------------------------------------------

# Input numbers
a = int(input())
b = int(input())
c = int(input())

# Count odd numbers
odd = (a % 2) + (b % 2) + (c % 2)

# Output
print(f"even {3 - odd}")
print(f"odd {odd}")
